lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -132,6 +132,12 @@ tokenize([], EndLine, _Column, #elixir_tokenizer{terminators=[{Start, {StartLine
   Message = io_lib:format("missing terminator: ~ts (for \"~ts\" starting at line ~B)", [End, Start, StartLine]),
   {error, {EndLine, Message, []}, [], Tokens};
 
+% ## VC merge conflict
```

Comment:
I think  this `##` is unnecessary, and I assume we should remove it from the other comments as well.
If we want emphasis it'd be better to use `%%` instead, `% ##` combo looks strange to me.